src/JPEGView/SettingsProvider.cpp

```diff
@@ -3,7 +3,7 @@
 #include "NLS.h"
 #include <float.h>
 #include <shlobj.h>
-#include <algorithm>
+#include <sstream>

```

Comment:
just wondering: what new call uses `sstream`?

Reply:
Sorry, I forgot to remove that. I was experimenting with parsing the config file but it's not needed anymore.

Reply:
> just wondering: what new call uses `sstream`?

Oh, I was wrong. Apparently it's needed for the 'stod' function.